ansible_collections/arista/avd/roles/eos_cli_config_gen/README.md

```diff
@@ -186,6 +187,16 @@ Requirements are located here: [avd-requirements](../../README.md#Requirements)
 
 #### IP Extended Access-Lists
 
+AVD currently supports 2 different data models for extended ACLs:
+
+- Legacy (for compatibility with existing designs, to avoid breaking changes)
+- Improved extended IP ACL data model
+
+Both data models can coexists without conflicts, as differet keys are used: `access_lists` vs `ip_access_lists`.
+Access list names must be unique.
+
+Legacy data model is supports simplified ACL definition with `sequence_number` -> `action_string`:
```

Comment:
Please rephrase this sentence. It is a bit hard to understand. Also, ```is supports``` doesn't sound good.